e2e/fixtures/rsc-router/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
```

Comment:
Wow, I thought the router one was doing things without index.html. Thank you so much.